# glueck2016phenoblocks

## PhenoBlocks: Phenotype Comparison Visualizations

<Paper>

::: slot concept

Figure 1 shows PhenoBlocks, a novel visual analytics tool that supports the comparison of phenotypes between patients built by Glueck et al. [^PhenoBlocks]. Phenotypic descriptions are often inconsistent, incomplete or imprecise [^Precision], PhenoBlocks addressees this via a differential hierarchy comparison algorithm that analyzes phenotypes pairwise between patients and display the results using Sunburst radial hierarchy layouts.

Glueck et al. develop PhenoLines with 2 clinical diagnosticians, together they reviewed 8,000 cases and concluded a diagnosis workflow which then informed the development of PhenoBlocks. Figure 3 shows a set of glyphs developed together with experts to represent different phenotypic abnormalities.

![](https://share.henry.wang/wEGjhn/MZFNgnoYup+)

:::

::: slot implementation

The authors propose a variant of Sunburst chart called small-multiple as shown in Figure 2, each small-multiple includes a hierarchy of an undiagnosed patient to a diagnosed reference patient. In the hierarchy, green indicates overlapping phenotypes in both patients, purple indicates missing phenotypes in the undiagnosed patient but present in the reference patient, and orange indicates missing phenotypes in the diagnosed reference patient.

<div class="even">
<p>

![](https://share.henry.wang/MczCDO/LSjrjIuj3y+)

![](https://share.henry.wang/qFQRiN/25hYIGcNWD+)

</p>
</div>

The overview of PhenoBlocks is shown in Figure 1. Section A consists of multiple small-multiples, each small-multiple has a hard handle score, which indicates the intensity of the phenotype, and a phenotype frequency as shown in Section C. Hovering over a small-multiple will present an enlarged small-multiple in Section B, hovering over a node in Section B will display a list of ancestors and descendants of the focused phenotype as shown in Section D.

Section E shows a legend, Section F shows the toolbar for user options and Section G shows the original data.

:::

::: slot data

- Data source: FORGE Canada Consortium
- Size: 47 patients, 6 genetic disorders
- Spatial dimensionality: 2D
- Temporal dimensionality: static
- Type: multivariate

:::

::: slot technique

- Sunburst chart
- Glyph

:::

::: slot related

Glueck et al. review previous work on relevant implementations for gene variant analysis such as Savant2 [^Savant], visualization of genome sequence ABySS [^ABySS], and visualization of genome comparison systems Circos [^Circos] and MizBee [^MizBee].

However, these systems do not focus on visualizing patient phenotypes, PhenoBlocks aims to visualize a specific patient's phenotype data against a cohort in order to inform subsequent analysis. The algorithm used for phenotype similarity comparison is inspired by PhenoTips [^PhenoTips] and Phenomizer [^Phenomizer].

:::

::: slot cited

60

:::

::: slot years

1984-2014

:::

::: slot domain

- Electronic Health Record Visualization
- Medical focus: Human Phenotype Ontology - Differential Hierarchy Comparison

:::

</Paper>

[^PhenoBlocks]: Glueck, M., Hamilton, P., Chevalier, F., Breslav, S., Khan, A., Wigdor, D., & Brudno, M. (2016). PhenoBlocks: Phenotype Comparison Visualizations. IEEE Transactions on Visualization and Computer Graphics, 22(1), 101–110. https://doi.org/10.1109/TVCG.2015.2467733

[^Precision]: Robinson, P. N. (2012). Deep phenotyping for precision medicine. Human Mutation. https://doi.org/10.1002/humu.22080

[^Savant]: Fiume, M., Smith, E. J. M., Brook, A., Strbenac, D., Turner, B., Mezlini, A. M., … Brudno, M. (2012). Savant Genome Browser 2: Visualization and analysis for population-scale genomics. Nucleic Acids Research. https://doi.org/10.1093/nar/gks427

[^ABySS]: Nielsen, C. B., Jackman, S. D., Birol, I., & Jones, S. J. M. (2009). ABySS-explorer: Visualizing genome sequence assemblies. IEEE Transactions on Visualization and Computer Graphics. https://doi.org/10.1109/TVCG.2009.116

[^Circos]: Krzywinski, M., Schein, J., Birol, I., Connors, J., Gascoyne, R., Horsman, D., … Marra, M. A. (2009). Circos: An information aesthetic for comparative genomics. Genome Research. https://doi.org/10.1101/gr.092759.109

[^MizBee]: Meyer, M., Munzner, T., & Pfister, H. (2009). MizBee: A multiscale synteny browsers. IEEE Transactions on Visualization and Computer Graphics. https://doi.org/10.1109/TVCG.2009.167

[^PhenoTips]: Girdea, M., Dumitriu, S., Fiume, M., Bowdin, S., Boycott, K. M., Chénier, S., … Brudno, M. (2013). PhenoTips: Patient phenotyping software for clinical and research use. Human Mutation. https://doi.org/10.1002/humu.22347

[^Phenomizer]: Köhler, S., Schulz, M. H., Krawitz, P., Bauer, S., Dölken, S., Ott, C. E., … Robinson, P. N. (2009). Clinical Diagnostics in Human Genetics with Semantic Similarity Searches in Ontologies. American Journal of Human Genetics. https://doi.org/10.1016/j.ajhg.2009.09.003