# glueck2017phenostacks

## PhenoStacks: Cross-Sectional Cohort Phenotype Comparison Visualizations

<Paper>

::: slot concept

Figure 1 shows PhenoStacks, a visualization system for cross-sectional phenotype comparison built by Glueck et al., it aims to support the exploration of phenotype variation within and between cross-sectional patient cohorts. The authors work with four genetics researchers for design, implementation and testing.

Phenotypes are observable and measurable patient traits primarily caused by genetic variation[^PhenoStacks]. Analyses of phenotypes can help understand how phenotype vary across patients with genetic diseases, both within and between cohorts.

![](https://share.henry.wang/fMaTKc/bMfzmHnbRz+)

:::

::: slot implementation

The authors interview genetics researchers and design an algorithm to simplify ontology topologies for visualization. Common tasks performed by the researchers can be easily done using PhenoStacks. The authors also use an "opt-out" approach which performs all the tasks and presents the results by default.

Figure 1 shows an overview of PhenoStacks. Section A shows a sunburst chart that visualizes summarized global patterns. Section B and C show groups of phenotypes of the selected ontology. Section D shows a matrix mapping patients in a cohort with phenotypes in Section C. Section E shows a search bar for searching the results.

Data preparation is done using Python with a data model based on the **cohort subgraph** shown in Figure 2 and Figure 3.

<div class="even">
<p>

![](https://share.henry.wang/36XTJY/debw4R79nQ+)

![](https://share.henry.wang/KBvhUC/hk17E8jsRj+)

</p>
</div>

:::

::: slot data

- Data source: two genetics researchers
- Size: 20-30 patients per researcher
- Spatial dimensionality: 2D
- Temporal dimensionality: static
- Type: multivariate

:::

::: slot technique

- Glyph
- Sunburst chart
- Matrix plot

:::

::: slot related

Glueck et al. identify **Visualization of Genetics Data** as the first relevant area in developing PhenoStacks, existing systems such as cBio[^cBio] and CircleMap[^CircleMap] use a compact color-coded matrix to represent genomic values. Savant 2[^Savant], Circos [^Circos] and MizBee [^MizBee] use space-filling layouts based on genome coordinates. Cytoscape [^Cytoscape] and VisANT [^VisANT] use network representations for visualizing gene pathways.

The second relevant area is **Visualizations of Cohorts**, the authors review a survey on visualization systems for electronic health records by Rind et al. [^Rind]. The authors also review COQUITO[^COQUITO] that analyzes temporal constraints of cohort membership. CAVA[^CAVA] explores temporal events between cohorts and CoCo[^CoCo] compares temporal events between cohorts.

Ontologies are used to capture the conceptual structure of a domain, **Visualizations of Ontologies** as the third relevant area is critical to establish a common ground for knowledge sharing, this is primarily built on a survey of ontology visualization methods by Katifori et al.[^Katifori].

:::

::: slot cited

45

:::

::: slot years

2003-2016

:::

::: slot domain

- Electronic Health Record Visualization
- Medical focus: Medical Genetics - Phenotypes

:::

</Paper>

[^PhenoStacks]: Glueck, M., Gvozdik, A., Chevalier, F., Khan, A., Brudno, M., & Wigdor, D. (2017). PhenoStacks: Cross-Sectional Cohort Phenotype Comparison Visualizations. IEEE Transactions on Visualization and Computer Graphics, 23(1), 191–200. https://doi.org/10.1109/TVCG.2016.2598469

[^cBio]: Cerami, E., Gao, J., Dogrusoz, U., Gross, B. E., Sumer, S. O., Aksoy, B. A., … Schultz, N. (2012). The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discovery. https://doi.org/10.1158/2159-8290.CD-12-0095

[^CircleMap]: Vaske, C. J., Benz, S. C., Sanborn, J. Z., Earl, D., Szeto, C., Zhu, J., … Stuart, J. M. (2010). Inference of patient-specific pathway activities from multi-dimensional cancer genomics data using PARADIGM. Bioinformatics. https://doi.org/10.1093/bioinformatics/btq182

[^Savant]: Fiume, M., Smith, E. J. M., Brook, A., Strbenac, D., Turner, B., Mezlini, A. M., … Brudno, M. (2012). Savant Genome Browser 2: Visualization and analysis for population-scale genomics. Nucleic Acids Research. https://doi.org/10.1093/nar/gks427

[^Circos]: Krzywinski, M., Schein, J., Birol, I., Connors, J., Gascoyne, R., Horsman, D., … Marra, M. A. (2009). Circos: An information aesthetic for comparative genomics. Genome Research. https://doi.org/10.1101/gr.092759.109

[^MizBee]: Meyer, M., Munzner, T., & Pfister, H. (2009). MizBee: A multiscale synteny browsers. IEEE Transactions on Visualization and Computer Graphics. https://doi.org/10.1109/TVCG.2009.167

[^Cytoscape]: Shannon, P., Markiel, A., Ozier, O., Baliga, N. S., Wang, J. T., Ramage, D., … Ideker, T. (2003). Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Research. https://doi.org/10.1101/gr.1239303

[^VisANT]: Hu, Z., Hung, J. H., Wang, Y., Chang, Y. C., Huang, C. L., Huyck, M., & Delisi, C. (2009). VisANT 3.5: Multi-scale network visualization, analysis and inference based on the gene ontology. Nucleic Acids Research. https://doi.org/10.1093/nar/gkp406

[^Rind]: Rind, A., Wang, T. D., Aigner, W., Miksch, S., Wongsuphasawat, K., Plaisant, C., & Shneiderman, B. (2011). Interactive Information Visualization to Explore and Query Electronic Health Records. Foundations and Trends® in Human–Computer Interaction, 5(3), 207–298. https://doi.org/10.1561/1100000039

[^COQUITO]: Krause, J., Perer, A., & Stavropoulos, H. (2016). Supporting Iterative Cohort Construction with Visual Temporal Queries. IEEE Transactions on Visualization and Computer Graphics. https://doi.org/10.1109/TVCG.2015.2467622

[^CAVA]: Zhang, Z., Gotz, D., & Perer, A. (2015). Iterative cohort analysis and exploration. Information Visualization. https://doi.org/10.1177/1473871614526077

[^CoCo]: Malik, S., Du, F., Monroe, M., Onukwugha, E., Plaisant, C., & Shneiderman, B. (2015). Cohort Comparison of Event Sequences with Balanced Integration of Visual Analytics and Statistics. Proceedings of the 20th International Conference on Intelligent User Interfaces - IUI ’15. https://doi.org/10.1145/2678025.2701407

[^Katifori]: Katifori, A., Halatsis, C., Lepouras, G., Vassilakis, C., & Giannopoulou, E. (2007). Ontology visualization methods---a survey. ACM Computing Surveys. https://doi.org/10.1145/1287620.1287621
